python/autocrop.py

```diff
@@ -1,52 +1,81 @@
-#!/usr/bin/env python
+#! /usr/bin/python3
```

Comment:
Let's not change this. This can sometimes cause problems when executing the script under a virtualenv, while the old one should work fine in such cases.